pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -44,6 +44,8 @@ type runtimeService interface {
 
 type policyName string
 
+var cpuManagerStateFilePath = "/var/lib/kubelet/cpu_manager_state"
```

Comment:
s/var/const

Is is possible to call this something like `DefaultCPUManagerStateFilePath` and pass it to `NewManager()` instead of referencing the global? I'd prefer to avoid mutating this global variable in tests. I apologize for not communicating clearly in the last review round.

Reply:
ok